client/src/components/NotifsDrawer.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from "react";
+import {
+  Drawer,
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+  ListItemAvatar,
+  ListItemIcon,
+} from "@material-ui/core";
+import ChatIcon from "@material-ui/icons/Chat";
+import MonetizationOnIcon from "@material-ui/icons/MonetizationOn";
+import { Link } from "react-router-dom";
+import notificationsAPI from "../notificationsAPI";
+
+const NotifsDrawer = ({
+  notifsOpen,
+  setNotifsOpen,
+  classes,
+  setUnreadCount,
+}) => {
+  const [notifs, setNotifs] = useState([]);
+
+  useEffect(() => {
+    setNotifs(notificationsAPI());
+  }, []);
+
+  useEffect(() => {
+    setUnreadCount(notifs.length);
+  }, [notifs, setUnreadCount]);
+
+  const notificationClickHandler = (notif) => {
+    notifs.splice(
+      notifs.findIndex((n) => n._id === notif._id),
+      1
+    );
+    setNotifsOpen(false);
+  };
+
+  return (
+    <Drawer
+      variant="temporary"
+      anchor="right"
+      open={notifsOpen}
+      onClose={() => setNotifsOpen(false)}
+    >
+      <div className={classes.drawerDiv} style={{ width: 350 }}>
+        <List component="nav" aria-label="navigation">
+          <ListItem divider style={{ height: 65, textAlign: "center" }}>
+            <ListItemText primary="NOTIFICATIONS" />
```

Comment:
For all caps text, it's better to have the literal text be normal case and use a [textTransform](https://www.w3schools.com/cssref/pr_text_text-transform.asp) to make it all-caps. To do it on a ListItemText you could use className or try some [primaryTypographyProps](https://material-ui.com/api/list-item-text/#props)

Though I know this is a lot of trouble for something that won't change it visually at all 🤷‍♀️

Reply:
Very doable 👍